app/src/main/java/de/westnordost/streetcomplete/quests/fire_hydrant/AddFireHydrantPosition.kt

```diff
@@ -0,0 +1,30 @@
+package de.westnordost.streetcomplete.quests.fire_hydrant
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.edits.update_tags.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement
+
+class AddFireHydrantPosition : OsmFilterQuestType<FireHydrantPosition>() {
+
+    override val elementFilter = """
+        nodes with
+         emergency = fire_hydrant and
+         (!fire_hydrant:position or fire_hydrant:position ~ "\?|fixme")
+         fire_hydrant:type and
+         (fire_hydrant:type = pillar or fire_hydrant:type = underground)
```

Comment:
This can probably be simplified:

```suggestion
         (fire_hydrant:type = pillar or fire_hydrant:type = underground)
```